core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -947,6 +947,16 @@ public int getNumberOfNoDictSortColumns() {
     return numberOfNoDictSortColumns;
   }
 
+  public CarbonColumn getRangeColumn() {
+    String rangeColumn =
+        tableInfo.getFactTable().getTableProperties().get(CarbonCommonConstants.RANGE_COLUMN);
+    if (rangeColumn == null) {
```

Comment:
Does this branch need to judge that rangeColumn is ""

Reply:
yes, create table command and set property will do it.
"" is invalid.